apps/src/storage/getColumnDropdown.js

```diff
@@ -39,10 +40,10 @@ export function getTables() {
 
 function getTableNameFromColumnSocket(socket, editor) {
   const paramValue = getFirstParam('getColumn', socket.parent, editor);
-  // The socket value has an extra set of double quotes. Trim off the first
-  // and last characters to remove, but don't use utils.stripQuotes because
-  // there may be other quotes in the table name (for example, apostrophes)
-  return paramValue.substring(1, paramValue.length - 1);
+
+  // The socket value (ex. the table name) has an extra set of double quotes in Droplet mode but not in text mode
+  // N.B. formatParamString is what removes the double quotes before we get here in text mode
```

Comment:
what does N.B. stand for?

Reply:
Here's the Wikipedia article: https://en.wikipedia.org/wiki/Nota_bene. This is something I learned in math classes and high school and college and thought was not that niche, but it looks like it is pretty niche and I'll just change it to `Note:` 😂